package.json

```diff
@@ -22,12 +22,11 @@
 	"devDependencies": {
 		"@commitlint/cli": "^9.1.2",
 		"@commitlint/config-conventional": "^9.1.2",
-		"@skyra/eslint-config": "^5.3.1",
-		"@types/node": "^14.6.1",
-		"@typescript-eslint/eslint-plugin": "^3.10.1",
-		"@typescript-eslint/parser": "^3.10.1",
-		"cz-conventional-changelog": "^3.3.0",
-		"discord.js": "github:discordjs/discord.js#fe7df708e44e0280dfaf0f8e457b154781bb5140",
```

Comment:
Please restore this as a development dependency, otherwise it won't installed with `yarn`.

Reply:
resolved in 0412b14